iroha/src/init.rs

```diff
@@ -45,22 +23,37 @@ pub mod config {
     use serde::Deserialize;
     use std::env;
 
-    const ROOT_PUBLIC_KEY: &str = "IROHA_ROOT_PUBLIC_KEY";
+    const PUBLIC_KEY: &str = "IROHA_INITIAL_PUBLIC_KEY";
+    const USER_NAME: &str = "IROHA_INITIAL_USERNAME";
+    const DOMAIN_NAME: &str = "IROHA_INITIAL_DOMAIN";
 
     #[derive(Clone, Deserialize, Debug)]
     #[serde(rename_all = "UPPERCASE")]
     pub struct InitConfiguration {
-        /// The public key of an initial "root@global" account
-        pub root_public_key: PublicKey,
+        /// The user name of initial account
+        pub user_name: String,
+        /// The domain of initial account
+        pub domain_name: String,
+        /// The public key of an initial account
+        pub public_key: PublicKey,
```

Comment:
Now that we have genesis we don't need any initial accounts apart from the genesis one which is added as part of `GenesisDomain`